libexec/bats-exec-test

```diff
@@ -68,6 +71,10 @@ load() {
   source "${filename}"
 }
 
+# timeout from here: http://stackoverflow.com/questions/3504945/timeout-command-on-mac-os-x
+# in case 'timeout' command does not exist on the system
+function timeout() { perl -e 'alarm shift; exec @ARGV' "$@"; }
```

Comment:
I like the idea of having a perl command fallback, for vanilla mac systems, especially since it requires no special modules.  For testing on Mac, you can "brew install coreutils" and add the path to the end of the PATH var like so `export PATH="$PATH:/usr/local/opt/coreutils/libexec/gnubin` (so it doesn't override the mac standard utils) or put it in front if you want them overridden.

There's ways to do this with vanilla BASH too, if you run the command in a background process, but it would be much trickier to work out.

Feel free to do a merge after any changes you make.